MailLibrary/Mail.php

```diff
@@ -78,7 +81,8 @@ public function __construct(Connection $connection, Mailbox $mailbox, $id)
 	public function __isset($name)
 	{
 		$this->headers !== NULL || $this->initializeHeaders();
-		return isset($this->headers[$this->formatHeaderName($name)]);
+		$key = $this->normalizeHeaderName($this->lowerCamelCaseToHeaderName($name));
```

Comment:
This is proposal. Basic idea is that magic methods uses magic name translation into normalized header name and getHeader() is more accurate (less magic included).